app/controllers/api/v1/application_controller.rb

```diff
@@ -65,3 +65,40 @@ def resource_name
     @resource_name ||= self.controller_name.singularize
   end
 end
+
+class ActionController::Parameters
+  def permit(params)
+    filter(params, self)
+  end
+
```

Comment:
This permit method is not patching an incorrect behaviour already present in Rails, but creating a new different implementation (with light logic inside). I suppose it's because the Rails checks are slow for the purpose of the api you are using.

Could this be done without modifying Rails? Eg: a new class that inherits from ActionController::Parameters.
So, in future, anybody can still use the rails default implementation.

I mean, because for example this definition changes the way the class works (in the rails implementation an exception is raised if the param is not present):

https://github.com/rails/rails/blob/master/actionpack/lib/action_controller/metal/strong_parameters.rb#L396